packages/xo-web/src/xo-app/host/tab-advanced.js

```diff
@@ -231,18 +230,18 @@ export default class extends Component {
             {host.enabled ? (
               <TabButton
                 btnStyle='warning'
-                handler={disableHost}
+                handler={maintenanceMode}
                 handlerParam={host}
                 icon='host-disable'
-                labelId='disableHostLabel'
+                labelId='maintenanceModeOn'
```

Comment:
Enabled host means that the maintenance mode is off. So, the message can be `Enable maintenance mode`.